### PR TITLE
initial draft

### PR DESCRIPTION
**Pull request by ipc103**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,92 @@
...
+## Objectives
+
+- Describe $resource
+- Use $resource methods
+- Fetch data from an API
+- Update data from an API
+
+## What is $resource?
+
+$resource is a factory that creates a resource object for us to communicate with APIs. We pass through a URL and it gives us an object back that looks like this:
```
Comment (ipc103) [inline README.md]: At this point, we haven't defined what a factory is. Can this be added to a previous README?

File: README.md [hunk 2]
```diff
@@ -1 +1,92 @@
...
+
+```js
+var User = $resource('/user/:userId');
+
+/**
+ * User = { get: function (),
```
Comment (ipc103) [inline README.md:22]: I like that you show what gets created for us here - this gives the user a peek at what `$resource` is doing for us.

**Comment by ipc103** [comment]
The presentation of how we'll use the `$resource` for our CRUD actions here are very clear, but the `$resource` is very abstract and will be difficult for students to understand. Before showing them this, can you show them how they would built out similar functionality using `$http`? After that, show them how awesome their lives will be if they can use the `$resource` object.

File: README.md [hunk 3]
```diff
@@ -1 +1,92 @@
...
+- Use $resource methods
+- Fetch data from an API
+- Update data from an API
+
+## What is $resource?
+
+$resource is a factory that creates a resource object for us to communicate with APIs. We pass through a URL and it gives us an object back that looks like this:
+
+```js
+var User = $resource('/user/:userId');
```
Comment (ipc103) [inline README.md:19]: is this the code that we'd put in our `services`? Can you show a more explicit example of how we'll use the `$resource` in our services? The next lab will be too difficult without seeing an example here.

File: README.md [hunk 4]
```diff
@@ -1 +1,92 @@
...
+ */
+```
+
+Here, we've defined our URL. Let's break that down:
+
+`/user/:userId`
+
+We're defining our URL as `/user/:userId` - but hold on, we can't have colons in our URLs!?
+
+Well, with $resource, using a colon in a URL means that section is *actually* a variable. `:userId` means that we aren't actually going to put `:userId` into our URL, rather that we will be replacing `:userId` with an actual user ID. This allows us to put our data into our URL - for instance, we would end up querying `/user/1` or `/user/1231938`.
```
Comment (ipc103) [inline README.md]: This makes sense - can you add a little bit more detail about what the `$resource` is inferring from that URL - how does it know where to make the proper get, post, and delete requests?

**Comment by AnnJohn** [comment]
@ipc103 could you check to see if this Readme was updated based on the comments on the subsequent lab? Thanks

**Pull request merged**